classification/classification_network.py

```diff
@@ -140,16 +131,29 @@ def createSensorRegion(network, sensorType, encoders, dataSource):
 
   @param dataSource   (RecordStream)  Sensor region reads data from here.
 
+  @param numCats      (int)           Number of possible categories per record.
+
   @return             (Region)        Sensor region of the network.
   """
   # Sensor region may be non-standard, so add custom region class to the network
-  if sensorType.split(".")[1] not in PY_REGIONS:
+  SENSOR_NAME = sensorType.split(".")[1]
+  SENSOR_MODULE = SENSOR_NAME  # conveniently have the same name
```

Comment:
It's a minor detail, but is there a specific reason why you decided to go with upper case names for SENSOR_NAME and SENSOR_MODULE? this is usually used for global module variables or constants. Just curious.